components/authz-service/storage/v2/project_test.go

```diff
@@ -36,7 +36,8 @@ func TestNewProject(t *testing.T) {
 		},
 	} {
 		t.Run(name, func(t *testing.T) {
-			if _, err := storage.NewProject(tc.id, tc.name, tc.typeVal); tc.expectErr != (err != nil) {
+			if _, err := storage.NewProject(
+				tc.id, tc.name, tc.typeVal, 0); tc.expectErr != (err != nil) {
```

Comment:
Funky indentation. There's no strong reason to worry about the scope here (what you'd usually prefer this constract for), so let's keep it simple
```go
_, err :=  storage.NewProject(tc.id, tc.name, tc.typeVal, 0); // falso: fix me! What's 0?
if tc.expectErr != (err != nil) {
	t.Fail()
}
```